centreon/tests/php/App/Upgrade/Infrastructure/Double/FakeUpdateRepository.php

```
@@ -0,0 +1,51 @@
+
+    public function findCurrentVersion(): ?string
+    {
+        return $this->currentVersion;
+    }
+
+    public function runUpdate(string $version): void
+    {
+        $this->updatesRun[] = $version;
+    }
+
+    public function runPostUpdate(string $currentVersion): void
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$currentVersion'. (undefined)

(UnusedFormalParameter)